src/OpenTK/Input/GamePadConfiguration.cs

```diff
@@ -172,25 +184,42 @@ static GamePadConfigurationSource ParseSource(string item)
             }
         }
 
-        static JoystickAxis ParseAxis(string item)
+        /// <summary>
+        /// Parses a string in the format a#" where:
+        /// - # is a zero-based integer number
+        /// </summary>
+        /// <param name="item">The string to parse</param>
+        /// <returns>The index of the axis</returns>
+        static int ParseAxis(string item)
         {
             // item is in the format "a#" where # a zero-based integer number
-            JoystickAxis axis = JoystickAxis.Axis0;
+            int axis = 0;
```

Comment:
This line is really not necessary anymore. We're not incrementing an enum, so it can be simplified.
On top of this, that change would make the method identical to ParseButton. It should be merged into a ParseIndex(string item) method.